src/components/pages/AdminPanel.js

```diff
@@ -27,11 +27,16 @@ const AdminPanel = (props) => {
 
 
 	const changeRole = (evt, username) => {
-		console.log(evt.target.value, username)
-		var role = evt.target.value
-		changeAccountRole(username, role, user.jwt)
-		.then(resp => 
-			console.log("Changed Roles"))
+		if(username === "admin@venus.com"){
+			alert("Admin user is not permitted to change roles.")
```

Comment:
small nitpick, but maybe should specify that the admin user is not permitted to change their own role, the current error message could be inferred as the admin user is not permitted to change any users roles.

Reply:
changed it to "Admin user is not permitted to change its own role."